pegasus/helpers/hoc_helpers.rb

```diff
@@ -56,7 +104,6 @@ def complete_tutorial(tutorial={})
         finished_at: DateTime.now,
         finished_ip: request.ip,
```

Comment:
As discussed on HipChat, this double counts if we are using the default weight of one. We should be putting them in the DB with a weight of zero.